packages/orbit-components/src/Alert/index.tsx

```diff
@@ -159,6 +159,7 @@ const Alert = (props: Props) => {
           title && inlineActions ? "flex-row" : "flex-col",
           !title && "items-center",
           inlineActions && "justify-between",
+          inlineActions && closable && "mr-md",
```

Comment:
This won't work well in RTL. Also, I'm wondering if we can't refactor the current _hack_ we are using for positioning the button (it being absolute with translates is 🤯 )… Can you take a look at that?

And yes, feel free to add more visual tests, for sure!

Reply:
> This won't work well in RTL.

Should be fixed now.

> Also, I'm wondering if we can't refactor the current hack we are using for positioning the button (it being absolute with translates is 🤯 )… Can you take a look at that?

Sure, I have some ideas. On it. 💻 


> And yes, feel free to add more visual tests, for sure!

OK. 👍

Reply:
Change with `absolute` position removal - https://github.com/kiwicom/orbit/pull/4435/commits/deba6bde63c532427ec4635c77c430f67e2735f9
Please, ignore the `min-h-icon-medium` classname changes.

<img width="1278" alt="image" src="https://github.com/user-attachments/assets/6995251e-f75b-473d-9e7c-9df1e6467035">

- checked with RTL version ✅ 

<img width="1289" alt="image" src="https://github.com/user-attachments/assets/db76f3c2-250b-4977-8574-06d0b0d0eb5f">

- checked without button and multiline content ✅ 
<img width="1289" alt="image" src="https://github.com/user-attachments/assets/9c9e04da-194d-4197-8fcf-302840dc79f5">

- mobile view ✅ 
<img width="392" alt="image" src="https://github.com/user-attachments/assets/935efc60-d81d-4b81-8e0f-e0d994fe3647">